setup.cfg

```diff
@@ -61,4 +61,4 @@ exclude_also =
     raise NotImplementedError
     raise RuntimeError
     \.\.\.
-    pass
+    pass$
```

Comment:
Is this intended?

Reply:
yeah, otherwise it was also skipping coverage checks on lines that contain the word  `password` or similar, which was wrong. So now it's only matching lines that end in `pass`, so that should cover the

```py
def foo():
    pass
```

cases